jupyterbook/_config.yml

```diff
@@ -0,0 +1,41 @@
+title: The U.S. Integrated Ocean Observing System (IOOS)
+author: IOOS 
+copyright: "2021"
+logo: images/IOOS-logo_little.png
+
+#only build files that are there in the table of contents. else defaults to building all files 
+only_build_toc_files: true
+
+launch_buttons:
+  notebook_interface: "classic" # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url: "https://mybinder.org"
+  colab_url: "https://colab.research.google.com"
+  thebe: true
+
+
+#######################################################################################
+# Execution settings
+execute:
+  execute_notebooks           : cache
```

Comment:
What exactly does this mean? The JB executes once and re-execute if there are changes?

Reply:
Yes. It caches all the notebook data into a folder. If there is any new change in the cells of the code/notebook, it re-builds the Jupyter{Book}.